dashboard/.env.development

```diff
@@ -1,4 +1,9 @@
+VITE_OIDC_ENABLED=true
 VITE_OIDC_AUTHORITY=http://keycloak:7470/realms/enduro
```

Comment:
Nitpick: I would consider changing the realm to "Artefactual" or something to indicate the realm is institutional level for a multi-tenant Keycloak instance. E.g. For the SSO trial we are creating a realm for the customer, and configuring our applications as clients in that realm.

Reply:
Good point, I called the realm "enduro" in https://github.com/artefactual-sdps/enduro/blob/main/hack/kube/components/dev/keycloak.yaml#L78, but we have clients for Temporal, MinIO and Enduro in that realm already. This is the default dev configuration, but we can change it ;)

Reply:
Yeah I thought about that after unfortunately... Sorry for not commenting on the previous PR. And yes - it's a nitpick because this is for dev work.